easy-ui-tokens/src/font/styles.json

```diff
@@ -0,0 +1,114 @@
+        "family": { "value": "{font.family.sans.value}" },
+        "weight": { "value": "{font.weight.semibold.value}" },
+        "size": { "value": "{font.size.lg.value}" },
+        "letter_spacing": {
+          "value": "{font.letter_spacing.widen_2.value}"
+        },
+        "line_height": { "value": "{font.line_height.normal.value}" }
+      },
+      "subtitle1": {
+        "family": { "value": "{font.family.sans.value}" },
+        "weight": { "value": "{font.weight.semibold.value}" },
+        "size": { "value": "{font.size.sm.value}" },
+        "letter_spacing": {
+          "value": "{font.letter_spacing.widen_2.value}"
+        },
+        "line_height": { "value": "{font.line_height.normal.value}" }
+      },
+      "subtitle2": {
+        "family": { "value": "{font.family.sans.value}" },
+        "weight": { "value": "{font.weight.medium.value}" },
+        "size": { "value": "{font.size.xs.value}" },
+        "letter_spacing": {
+          "value": "{font.letter_spacing.widen_1.value}"
+        },
+        "line_height": { "value": "{font.line_height.normal.value}" }
+      },
+      "body1": {
+        "family": { "value": "{font.family.sans.value}" },
+        "weight": { "value": "{font.weight.normal.value}" },
+        "size": { "value": "{font.size.sm.value}" },
+        "letter_spacing": {
+          "value": "{font.letter_spacing.widen_5.value}"
+        },
+        "line_height": { "value": "{font.line_height.normal.value}" }
+      },
+      "body2": {
+        "family": { "value": "{font.family.sans.value}" },
+        "weight": { "value": "{font.weight.normal.value}" },
+        "size": { "value": "{font.size.xs.value}" },
+        "letter_spacing": {
+          "value": "{font.letter_spacing.widen_3.value}"
+        },
+        "line_height": { "value": "{font.line_height.relaxed.value}" }
+      },
+      "caption": {
+        "family": { "value": "{font.family.sans.value}" },
+        "weight": { "value": "{font.weight.normal.value}" },
+        "size": { "value": "{font.size.2xs.value}" },
+        "letter_spacing": {
+          "value": "{font.letter_spacing.widen_4.value}"
+        },
+        "line_height": { "value": "{font.line_height.relaxed.value}" }
+      },
+      "overline": {
+        "family": { "value": "{font.family.sans.value}" },
+        "weight": { "value": "{font.weight.normal.value}" },
+        "size": { "value": "{font.size.3xs.value}" },
+        "letter_spacing": {
+          "value": "{font.letter_spacing.widen_6.value}"
+        },
+        "line_height": { "value": "{font.line_height.relaxed.value}" }
+      },
+      "button": {
+        "family": { "value": "{font.family.sans.value}" },
+        "weight": { "value": "{font.weight.medium.value}" },
+        "size": { "value": "{font.size.base.value}" },
+        "letter_spacing": {
+          "value": "{font.letter_spacing.normal.value}"
+        },
+        "line_height": { "value": "{font.line_height.normal.value}" }
+      }
```

Comment:
i removed some existing ones for now so that they could be revisited. some didn't seem too parallel with the others. we can bring them over when we make their respective components if we feel good about it at that time

Reply:
good call